tests/python/unittest/test_image.py

```diff
@@ -356,17 +356,18 @@ def test_det_augmenters(self):
             pass
 
     @with_seed()
-    @unittest.skip('Flaky test. Skipping until a solution can be found. Tracked by https://github.com/apache/incubator-mxnet/issues/14718')
     def test_random_size_crop(self):
         # test aspect ratio within bounds
         width = np.random.randint(100, 500)
         height = np.random.randint(100, 500)
         src = np.random.rand(height, width, 3) * 255.
         ratio = (0.75, 1)
+        epsilon = 0.05
```

Comment:
how do we decide epsilon? is 0.05 too large here?

Reply:
Thanks @stu1130 for the discussion offline. The randomness is caused by sqrt of area / or * ratio. Same epsilon is used in pytorch test as well. Approve this change thereby.